src/content/fr/2019/fonts.md

```diff
@@ -0,0 +1,682 @@
+
+## Les polices variables sont l'avenir
+
+[Les polices variables](https://developer.mozilla.org/fr/docs/Web/CSS/CSS_Fonts/Variable_Fonts_Guide) permettent d'inclure plusieurs poids et styles de police dans un seul fichier.
+
+{{ figure_markup(
+  caption="Pourcentage des pages qui incluent une police variable.",
+  content="1.8%",
+  classes="big-number"
+)
+}}
+
+Avec 1, 8 %, c'est plus que prévu, encore que je sois enthousiaste de voir ce décollage. <a hreflang="en" href="https://developers.google.com/fonts/docs/css2">Google Fonts v2</a> inclut un certain support pour les polices variables.
+
+{{ figure_markup(
+  image="fig19.png",
+  caption="Usage of 'font-variation-settings' axes.",
+  description="Graphique à barres montrant l'utilisation de la propriété font-variation-settings. 42% des propriétés des pages pour ordinateur de bureau sont réglées sur la valeur `opsz`, 32% sur `wght`, 16% sur `wdth`, 2% ou moins sur `roun`, `crsb`, `slnt`, `inln`, et plus. Les différences les plus notables entre les pages pour ordinateur de bureau et les pages mobiles sont l'utilisation à 26 % de `opsz`, 38 % de `wght` et 23 % de `wdth`.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQDogXDb3BwZZHrBT39qccP_LJoCScD3QEi_FmjT_8VDPD_1Srpz-g7ZuuTUEb8pYXBpDmQzZ1hQh7q/pubchart?oid=699343351&format=interactive"
+  )
+}}
+
+Dans l'optique de ce vaste ensemble de données, il s'agit d'échantillons de très petite taille - prenez ces résultats avec des pincettes. Cependant, l'axe `opsz`, qui est le plus courant sur les pages desktop, est notable, avec `wght` et `wdth` à la traîne. D'après mon expérience, les démos d'introduction aux polices variables sont généralement basées sur le poids.
+
+## Les polices de couleur pourraient-elles aussi être l'avenir ?
+
+{{ figure_markup(
+  caption="Nombre de pages pour ordinateurs de bureau incluant une police de couleur.",
+  content="117",
+  classes="big-number"
+)
+}}
+
+Leur utilisation est pratiquement inexistante ici, mais vous pouvez consulter l'excellente ressource <a hreflang="en" href="https://www.colorfonts.wtf/">Color Fonts ! WTF ?</a> pour plus d'informations. Semblable (mais pas du tout) au format SVG pour les polices (qui est mauvais et qui disparaît), cela vous permet d'intégrer le SVG dans les fichiers OpenType, ce qui est génial et cool.
+
+## Conclusion
+
+Le point le plus important ici est que Google Fonts domine la conversation sur les polices de caractères sur le web. Les approches qu'ils ont adoptées pèsent lourdement sur les données que nous avons enregistrées ici. Les points positifs sont un accès facile aux polices web, de bons formats de police (WOFF2), et des configurations gratuites de `unicode-range`. Les inconvénients sont les défauts de perfomrance liés à l'hébergement par des tiers, les requêtes vers des hôtes différents et l'absence d'accès au `preload`.
```

Comment:
`perfomrance`/`performance`

Reply:
Fixed in https://github.com/HTTPArchive/almanac.httparchive.org/commit/6df484a83f05c285020d15faa5a0207a42c519e7